src/Aspire.Cli/DotNet/DotNetCliRunner.cs

```diff
@@ -46,6 +46,19 @@ internal class DotNetCliRunner(ILogger<DotNetCliRunner> logger, IServiceProvider
 
     internal Func<int> GetCurrentProcessId { get; set; } = () => Environment.ProcessId;
 
+    internal Func<DateTime> GetCurrentProcessStartTime { get; set; } = () =>
```

Comment:
Instead of returning the date time, return a long in the form of unix epoch time.